rust/datafusion/src/execution/context.rs

```diff
@@ -195,43 +169,6 @@ impl ExecutionContext {
         &self.scalar_functions
     }
 
-    fn build_schema(&self, columns: &Vec<SQLColumnDef>) -> Result<Schema> {
```

Comment:
This code is just moved to https://github.com/apache/arrow/pull/7906/files#diff-21b952259e12c1adf2098d9a797caef1R134 in https://github.com/apache/arrow/pull/7906/commits/aaaa9b7da9d9566ca6f3fc84d7b272951ff51f40